installer/roles/image_build/files/supervisor.conf

```diff
@@ -46,8 +46,20 @@ stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
+[program:awx-rsyslogd]
+command = rsyslogd -n -i /var/run/rsyslog/rsyslog.pid -f /var/lib/awx/rsyslog/rsyslog.conf
```

Comment:
same as for the socket path, would rename the directory for the pid file as well.

Reply:
✅ @rooftopcellist let's track this `/var/run/` renaming as a task post-merge